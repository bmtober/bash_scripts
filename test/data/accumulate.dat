509.20
-119.28
-305.49
-70.38
0.01
488.67
