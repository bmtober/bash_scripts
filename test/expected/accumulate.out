   509.20	  509.20	  1
   389.92	 -119.28	  2
    84.43	 -305.49	  3
    14.05	  -70.38	  4
    14.06	    0.01	  5
   502.73	  488.67	  6
